R/tm_g_bivariate.R

```diff
@@ -535,6 +538,13 @@ srv_g_bivariate <- function(id,
       width = plot_width
     )
 
+    teal.widgets::verbatim_popup_srv(
+      id = "warnings",
+      reactive(paste(get_warnings(output_q()), collapse = "\n")),
+      title = "Warning",
+      disabled = reactive(is.null(teal.code::get_code(output_q())))
```

Comment:
So I do like the button being disabled if there's no warnings - I notice in other modules this isn't there